BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -53,6 +53,7 @@ class BraintreeClient @VisibleForTesting internal constructor(
 
     private val crashReporter: CrashReporter
     private var launchesBrowserSwitchAsNewTask: Boolean = false
+    private val deviceInspector: DeviceInspector
```

Comment:
I need the deviceInspector here because the client (PayPalClient) needs to know if PayPal is installed

https://github.com/braintree/braintree_android/blob/06fac45120d563630c55bf9ff95ff97230892e88/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt#L46

Reply:
Oh got it yeah. We could also move the `DeviceInspector` reference into `PayPalInternalClient` if that works? I'm hoping we'll be able to phase out `BraintreeClient` over time.